front-end/src/components/Dashboard/index.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import styled from 'styled-components';
+import SideBarNav from "../../shared/SideBarNav";
+import jwt_decode from 'jwt-decode'
+import axios from "axios";
+
+class Dashboard extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: '',
+      email: '',
+      isLoaded: false
+    };
+  }
+
+  componentDidMount() {
+    let token = localStorage.getItem('access_token');
+    let decoded = jwt_decode(token);
+    let userID = decoded.id;
+
+    axios.get('http://localhost:3000/users/'+ userID,{ headers:{
```

Comment:
Move this into the the api folder similar to Jed's login code
the error code in line 34 can be captured in a try catch block. Axios throws any HTTP status that are 4xx or 5xx

Reply:
I can take care of these tomorrow morning before the test and deploy. My changes had some refactoring that introduced merge conflicts so I can fix those quickly.